service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -195,7 +197,10 @@ internal class RevurderingServiceImpl(
                     return KunneIkkeRevurdere.KunneIkkeOppretteOppgave.left()
                 }
 
-                revurdering.tilAttestering(oppgaveId, saksbehandler)
+                oppgaveService.lukkOppgave(revurdering.oppgaveId).fold(
+                    ifLeft = { return KunneIkkeRevurdere.KunneIkkeLukkeOppgave.left() },
```

Comment:
Det er litt problematisk å returnere left her siden vi allerede har laget en ny oppgave. I de andre tilfellene har vi bare logget dette og fortsatt som alt er fint. Oppgaven må da lukkes manuelt. Enten av oss eller av en saksbehandler (men de vil ikke få beskjed slik som det er nå).

Reply:
Hehe ja det er sant, legger in litt logging så vi får meldt eventuella problem iallafall!